src/borgstore/backends/rclone.py

```diff
@@ -78,44 +79,52 @@ def __init__(self, path, *, do_fsync=False):
         self.user = "borg"
         self.password = secrets.token_urlsafe(32)
 
+    def find_open_port(self):
```

Comment:
not sure I understand this method.

does it find an **available** port, rather than an **open** port?

Reply:
I see how "open" might be a problem here. I'll rename the method to be "find_available_port".